Fakebook.Posts/Fakebook.Posts.UnitTests/Services/SpamServiceTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+using Fakebook.Posts.RestApi.Services;
+using Fakebook.Posts.Domain.Interfaces;
+using Fakebook.Posts.Domain.Models;
+using Moq;
+using Fakebook.Posts.DataAccess.Mappers;
+
+namespace Fakebook.Posts.UnitTests.Services
+{
+    public class SpamServiceTests
+    {
+
+        private readonly IPostsRepository _postRepository;
+        private readonly ITimeService _timeService;
+        private readonly TimeService timeService = new();
+
+        public string contentIsDuplicate = "this is an old post";
+        public string contentIsNotDuplicate = "testing a post";
+        public string oldPostContent = "this is an old post";
+        public string testUserEmail = "test.user@revature.net";
+        public int secondsTimeout = 10;
+
+        public DateTime dateNow;
+        public DateTime dateNotRecent = new DateTime(2021, 4, 4);
+        
+        public SpamServiceTests()
+        {
+            dateNow = timeService.CurrentTime;
+        }
+
+        [Fact]
+        public void CheckSpamService_IsNotSpammingByTime_IsTrue()
+        {
+       
+            Post post = new(testUserEmail, oldPostContent)
+            {
+                Id = 1,
+                Picture = "picture",
+                CreatedAt = dateNotRecent
+            };
+
+            CheckSpamService checkSpamService = new CheckSpamService(_postRepository, _timeService);
+            
+            List<Post> oldPosts = new List<Post>();
+            oldPosts.Add(post);
+
+            bool isNotSpam = checkSpamService.CheckTimeSpam(oldPosts, secondsTimeout, dateNow);
+            Assert.True(isNotSpam);
+        }
+
+        [Fact]
+        public void CheckSpamService_IsNotSpammingByTime_IsFalse()
+        {
+
```

Comment:
empty lines don't belong right next to `{` and `}`. but i'm not going to insist on it now